Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -656,6 +661,14 @@ private void RenderNodes (HtmlTextWriter writer, WebTreeNodeCollection nodes, bo
           RenderNodeChildren (writer, node, isLastNode, hasExpander, nodeIDAlgorithm, nestingDepth);
 
         writer.RenderEndTag(); // End node block
+
+        if(!EnableTopLevelGrouping) continue;
+        if (isLastNode || nestingDepth != 0) continue;
+        var nextNode = nodes[i + 1];
+        if (nodes[i].Category != nextNode.Category)
+        {
+          RenderSeparator (writer);
```

Comment:
We should use intelligent CSS instead of brute-forcing this. In DropDownMenu, it was done this way because this was done in 2005 and we didn't know any better (or it was cheaper to do at the time) but the resulting extra element means we always need to consider this when working with the DOM and that's not somehting we should continue to do in new code.

So, apply the css-class to the first or last node of the category (depending on which is the cleaner approach) and then use css classes and/or css-pseudo-elements to generate the splitter.